server/service/IndividualServicesService.js

```diff
@@ -100,18 +138,56 @@ exports.provideListOfConnectedDevices = function (user, originator, xCorrelator,
  * customerJourney String Holds information supporting customer’s journey to which the execution applies
  * returns inline_response_200_4
  **/
-exports.putLiveAirInterfacePerformanceMonitoringIsOn = function (body, mountName, uuid, localId, user, originator, xCorrelator, traceIndicator, customerJourney) {
-  return new Promise(function (resolve, reject) {
-    var examples = {};
-    examples['application/json'] = {
-      "response-code": 204
+exports.putLiveAirInterfacePerformanceMonitoringIsOn = async function (body, mountName, uuid, localId, user, originator, xCorrelator, traceIndicator, customerJourney) {
```

Comment:
Please replicate the same comment provided for the previous function to all the below functionalities.